index.html

```diff
@@ -4415,22 +4421,39 @@ <h3>JSON-LD</h3>
 
         <ul>
           <li>
-Use of JSON-LD keywords in the `@context` value that globally modify document
-value processing, such as global settings of `@base`
+In-line declaration of JSON-LD keywords in the `@context` value that globally
+modify document term and value processing, such as setting `@base` or `@vocab`,
           </li>
           <li>
-In-line declaration of JSON-LD contexts in the `@context` property.
+Use of JSON-LD contexts that override declarations in previous contexts, such as
+resetting `@vocab`,
+          </li>
+          <li>
+In-line declaration of JSON-LD contexts in the `@context` property, and
           </li>
           <li>
 Use of full URLs for JSON-LD terms and types (e.g.,
 `https://www.w3.org/2018/credentials#VerifiableCredential` or
 `https://vocab.example/myvocab#SomeNewType`) instead of the short forms of
 any such values (e.g., `VerifiableCredential` or `SomeNewType`) that are
-either explicitly defined as JSON-LD `@context` mappings (e.g.,
-`https://www.w3.org/ns/credentials/v2`) or are implicitly defined via the
-`@vocab` feature that applies to all undefined terms.
+explicitly defined as JSON-LD `@context` mappings (e.g.,
+`https://www.w3.org/ns/credentials/v2`).
           </li>
-       </ul>
+        </ul>
+
+        <p>
+While this specification warns against the usage of `@vocab`, there are
+legitimate usages of the feature, such as to ease experimentation, development,
+and localized deployment. If an application developer desires to use `@vocab` in
+production, which is strongly advised against, they are urged to understand that
```

Comment:
```suggestion
production, which is advised against, they are urged to understand that
```

perhaps something like 

```suggestion
production, which is advised against to reduce term collisions and leverage the benefits of semantic interoperability, they are urged to understand that
```

Reply:
When I pulled in @TallTed's change suggestions, it invalidated this one, can you please re-raise it on the updated text so you get credit for the change?